decidim-admin/config/locales/en.yml

```diff
@@ -135,6 +135,10 @@ en:
         file: File
     errors:
       models:
+        meeting:
```

Comment:
This needs to be added to `decidim-meetings/config/locales/en.yml` as it is related to the meetings.

Reply:
Thanks @alecslupu, this has now been updated.